### PR TITLE
Add "function coloring" to glossary

### PR DESCRIPTION
**Pull request by bakkot**

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

**Review by rkirsling — APPROVED** [review]
Thanks!!

File: terminology.md [hunk 1]
```diff
@@ -510,6 +510,21 @@ While this is not an explicitly adopted goal of TC39, it is a common standards c
 
 [HTML Design Principles](https://www.w3.org/TR/html-design-principles/#priority-of-constituencies)
 
+## Function coloring
+
+### Definition
```
Comment (ljharb) [inline terminology.md:515]:
```suggestion
### Definition

```

**Review by michaelficarra — APPROVED** [review]
Don't love the spelling of colouring, but LGTM.

**Review by syg — APPROVED** [review]
on god i will disapprove this if it's spelled "colouring"

**Pull request merged**